src/ls_instructions/ls_instructions_parse.cpp

```diff
@@ -127,6 +127,12 @@ LSInstruction parse_ls_instruction(std::string_view line)
             counter++;
         }
         return {BellPairInit{side1, side2, locs[0], locs[1]}};
+    }      
+    else if(instruction == "RequestYState" || instruction == "9")
+    {
+        auto patch_id1 = parse_patch_id(get_next_arg());
+        auto patch_id2 = parse_patch_id(get_next_arg());
+        return {YStateRequest{patch_id1, patch_id2}, YStateRequest::DEFAULT_WAIT};
```

Comment:
Could make the name consistent with the `YStateRequest` struct because someone will be looking at this code to figure out how to write LLI

Reply:
Are you referring to "RequestYState" in line 131? I was copying the convention where "RequestMagicState" returns a MagicStateRequest LLI. I can change both if you want.

Reply:
sorry, I meant the names of the variables patch_id1 and patch_id2